### DISCUSSION TITLE
Discoverability features

### DISCUSSION THREAD

**Discussion by meljoann** [funmusicplace/mirlo#193] [Ideas]
Coming from the Discord:

- Discoverability is a major pain point for indie musicians, Fedi has a lot of potential to ease this
- Mirlo could integrate with Fedi accounts, maybe with RSS, to "bookmark" music you've discovered there
- Bonus if there is a way to follow Fedi hashtags, or import a list of good-music-sharers 
- You could then create one or more "on-the-go" playlist, to put on when you feel like listening to new music. Or save entire albums to listen later

**Comment by meljoann** [comment]
Maybe something like:

Sign in with Fedi account > parse "bookmarks": does the post contain a Mirlo link? > yes = include in playlist feed

**Comment by Cwpute** [comment]
Instead of linking accounts - which always irks me a little bit i have to admit - i'd be much more cool with the ability to cross-post articles or playlist from Mirlo in other places on the Fediverse. In short, ot integrate the ActivityPub protocol for articles/playlists that may be implemented in Mirlo at some point :eyes: https://github.com/funmusicplace/mirlo/discussions/530

**Reply by robnsn** [reply]
ActivityPub / fediverse integration as a feature would be enough to induce me to switch to Mirlo.
